Sources/Core/Utilities/ImageSource/UrlImage.swift

```diff
@@ -16,6 +16,7 @@ class UrlImageModel: ObservableObject {
   
   init(url: URL?) {
     self.url = url
+    // TODO: Load from cache
```

Comment:
I would not try to replicate the whole Kingfisher logic. Let's just do without. So if dev decides to go without Kingfisher, he'll only have bare bone image loading mechanism.